st2tests/st2tests/config.py

```diff
@@ -29,7 +29,13 @@ def parse_args():
 
 def _setup_config_opts():
     cfg.CONF.reset()
-    _register_config_opts()
+
+    try:
+        _register_config_opts()
+    except Exception:
+        # Some scripts register the options themselves which means registering them again will
+        # cause a non-fatal exception
+        return
```

Comment:
Can you log the exception at debug level so that in case this is not expected we can get a clue from the logs. This is particularly because you are catching all exceptions.